templates/swift/Sources/OAuth/WebAuthComponent.swift.twig

```diff
@@ -53,17 +53,28 @@ public class WebAuthComponent {
     ///   - url: The URL containing the cookie
     ///
     public static func handleIncomingCookie(from url: URL) {
-        let components = URLComponents(string: url.absoluteString)!
 
-        let cookieParts = [String: String](uniqueKeysWithValues: components.queryItems!.map {
-            ($0.name, $0.value!)
+        guard let components = URLComponents(string: url.absoluteString),
+            let queryItems = components.queryItems else {
+            return
+        }
+        
+        let cookieParts = [String: String](uniqueKeysWithValues: queryItems.compactMap { item in
+            item.value.map { (item.name, $0) }
         })
 
-        var domain = cookieParts["domain"]!
+        guard var domain = cookieParts["domain"],
+              let key = cookieParts["key"],
+              let secret = cookieParts["secret"] else {
+            return
+        }
+    
+        domain.remove(at: domain.startIndex)
+        var domain = validatedCookieParts.domain
         domain.remove(at: domain.startIndex)
+        let key = validatedCookieParts.key
+        let secret = validatedCookieParts.secret
```

Comment:
We can remove these lines now

Reply:
ha.  a reminder to not use the github editor.